compiler/scenario-service/protos/scenario_service.proto

```diff
@@ -181,6 +181,11 @@ message ScenarioError {
     sfixed64 effective_at = 2;
   }
 
+  message SubmitterNotInMaintainers {
+    Party submitter = 1;
+    repeated Party maintainers = 2;
```

Comment:
For debugging purposes, I'd recommend to also mention the contract ID that caused the problem.

Reply:
:+1:

Reply:
done.